source/jsonnet/england-wales/ccs/blocks/who-lives-here/any_visitors.jsonnet

```diff
@@ -22,6 +22,37 @@ local placeholders = import '../../../lib/placeholders.libsonnet';
         },
       ],
     },
+    definitions: [
+      {
+        title: 'Electronic Showcard',
+        contents: [
+          {
+            description: 'Include',
+          },
+          {
+            list: [
+              'people who usually lived somewhere else in the UK, for example, boyfriends, girlfriends, friends or relatives',
+              'people staying because it was their second address, for example, for work - their permanent or family home was elsewhere',
+              'people who usually lived outside the UK who were staying in the UK for less than 3 months',
```

Comment:
`less than 3 months` should be wrapped in `<strong>` tags.

Reply:
Thanks👍